File: app/src/main/java/com/newchar/woolhelper/state/IStateLayout.java
package com.newchar.woolhelper.state;

/**
 * @author wenliqiang@100tal.com
 * date            2019-12-21
 * @since 当前版本描述，
 * @since 迭代版本描述
 */
public interface IStateLayout<T> {

   void attach(T attach);

   T getHost();

}

File: app/src/main/java/com/newchar/woolhelper/replacer/IStateLayout.kt
package com.newchar.woolhelper.replacer

/**
 * @author wenliqiang@100tal.com
 * date            2019-12-21
 * @since 当前版本描述，
 * @since 迭代版本描述
 */
interface IStateLayout<T> {
    fun attach(attach: T)
    val host: T
}